pmd-scala-modules/pmd-scala-common/src/test/resources/net/sourceforge/pmd/lang/scala/cpd/testdata/special_comments.txt

```diff
@@ -0,0 +1,52 @@
+    [Image] or [Truncated image[            Bcol      Ecol
+L1
+    [// Testing CPD suppression]            1         27
+L3
+    [// Irrelevant comment]                 1         22
+L4
+    [// CPD-OFF]                            1         11
```

Comment:
Shouldn't `CPD-OFF` tokens be filtered out too? In other languages I think that's how it works

Reply:
Comments are now skipped and no longer tokenised